EGAR-772 aircrafts pagination

nguyendhod opened the pull request:
**What changes does this PR bring?**
Brings pagination as a reusable set of template and javascript code, demonstrable for the manage aircrafts screen. This will need to be folded out into other screens, not least the aircrafts screen during GAR form entry, but this is to get it into staging for perusal.

This also includes what would be considered a temporary fix for DSUP-3469.

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [x] Have you built the code locally and confirmed its working?
- [x] Is the build confirmed to be passing on CI?
- [x] Have you added enough relevant unit tests for your change?
- [x] Have you added enough relevant E2E tests for your change?
- [x] Have you updated any relevant documentation as part of this change?
- [x] Has this change been tested against the Acceptance Criteria?
- [x] Have you considered how this will be deployed in SIT/Staging/Production?

chanc789 commented on src/common/services/personApi.js:
```diff
@@ -98,15 +98,17 @@ module.exports = {
    * @returns {Promise} resolves with API response
    */
-  getPeople(id, userType) {
+  getPeople(id, userType) { // , page) {
```
Remove this commented out code

nguyendhod replied:
lol, hangover from the attempt at people screen pagination. Will remove.

nguyendhod commented:
E2E tests have been merged into master ready for this, apologies.

Have addressed the comments, so if ok to merge, please do so, will review the other pull request.
